sortedcontainers/sorteddict.pyi

```diff
@@ -0,0 +1,103 @@
+import typing
+from typing import (
+    Any,
+    Callable,
+    Dict,
+    Generic,
+    Hashable,
+    Iterator,
+    Iterable,
+    ItemsView,
+    KeysView,
+    List,
+    Mapping,
+    Optional,
+    Sequence,
+    Type,
+    TypeVar,
+    Tuple,
+    Union,
+    ValuesView,
+)
+
+_T = TypeVar("_T")
+_S = TypeVar("_S")
+_KT = TypeVar("_KT", bound=Hashable)  # Key type.
+_VT = TypeVar("_VT")  # Value type.
+_KT_co = TypeVar("_KT_co", covariant=True, bound=Hashable)
+_VT_co = TypeVar("_VT_co", covariant=True)
+_SD = TypeVar("_SD", bound="SortedDict")
```

Comment:
`SortedDict` doesn't need quotes around it. In `pyi` files, forward references work without quotes.